app/exec/extension/_lib/merger.ts

```diff
@@ -107,6 +107,14 @@ export class Merger {
 				partials.forEach((partial) => {
 					if (_.isArray(partial["targets"])) {
 						targets = targets.concat(partial["targets"]);
+
+						if(targets.length > 0) {
```

Comment:
This is not where this should go. See [VSSIntegrationComposer](https://github.com/Microsoft/tfs-cli/blob/master/app/exec/extension/_lib/targets/Microsoft.VisualStudio.Services.Integration/composer.ts) for an example of how to do validation. I'd like to remove validation steps from this file and vsix-manifest-builder.